.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": ["eslint:recommended", "plugin:react/recommended", "prettier"],
```

Comment:
We can probably add more here, but for now, this is what `npx eslint --init` gave me. (Well, I added `prettier`.)